Formula/habitat.rb

```diff
@@ -0,0 +1,15 @@
+class Habitat < Formula
+  desc "Application automation framework to build applications"
+  homepage "https://www.habitat.sh/"
+
+  url "https://api.bintray.com/content/habitat/stable/darwin/x86_64/hab-0.6.0-20160613151520-x86_64-darwin.zip?bt_package=hab-x86_64-darwin"
```

Comment:
You'll want to go directly to the CDN link of:
`https://habitat.bintray.com/stable/darwin/x86_64/hab-<VERSION>-x86_64-darwin.zip`

The Bintray API link is only useful if you want to use a logical version of `$latest` in your URL like we do on the main [Habitat download page](https://www.habitat.sh/docs/get-habitat/): `https://api.bintray.com/content/habitat/stable/darwin/x86_64/hab-$latest-x86_64-darwin.zip?bt_package=hab-x86_64-darwin` (note the `$latest`).

You'll also want to use the latest 0.7.0 release in this initial formula:

URL -> `https://habitat.bintray.com/stable/darwin/x86_64/hab-0.7.0-20160614231131-x86_64-darwin.zip`
SHA256 -> `93fac880261df34c75ac5775c9febb536b82ac955cd6df873bbfec1cf697ed50`